File: osis/labs/LR-5-6/block-2/main.pas
program БАМБАМБИМБИМ БРР БРР ПАТАПИМ;

var
  a: array[1..10] of Integer;
  i, count, max_count: Integer;
begin
  { Заполняем массив случайными числами от 1 до 10 }
  count:=0;
  max_count:=0;
  for i := 1 to 10 do
  begin
    a[i] := Random(10) + 1;
  end;
  Writeln(a);
  
  for i:=1 to 9 do
  begin
    if a[i] >= a[i+1] then count := count + 1
    else begin
      count:= count + 1;
      if max_count < count then max_count:= count;
      count:=0
    end;
    end;
    Writeln(max_count)
  end.

File: osis/labs/LR-5-6/block-2/main.py
from random import randint

a = [randint(1, 10) for _ in range(10)]
print(a)
max_count = 0
count = 0
for i in range(len(a)-1):
    if a[i] >= a[i+1]:
        count += 1
    else:
        count += 1
        if max_count < count:
            max_count = count  
        count = 0          

print(max_count)